Sources/TokamakDOM/Views/SecureField.swift

```diff
@@ -29,10 +29,10 @@ extension SecureField: ViewDeferredToRenderer where Label == Text {
     ], listeners: [
       "keypress": { event in if event.key == "Enter" { proxy.onCommit() } },
       "input": { event in
-        if let newValue = event.target.object?.value.string {
-          proxy.textBinding.wrappedValue = newValue
-        }
-      },
+          if let newValue = event.target.object?.value.string {
+            proxy.textBinding.wrappedValue = newValue
+          }
+        },
```

Comment:
Sorry, for pestering you with formatting comments, but what version of SwiftFormat do you have installed? Is that the latest version? It seems like we're changing this area of code back and forth in our PRs because our different versions of SwiftFormat can't agree how it should look 😅

Reply:
I upgraded `swiftformat` and it fixed it.